### DISCUSSION TITLE
Merging issue

### DISCUSSION THREAD

**Discussion by samuel-404** [sanjay-kv/Open-source-Practice#1060] [Q&A] [answered]
how long does it on average to merge a pull request

**Comment by irfanbroo** [comment, marked as answer]
Hey, Samuel. On average it takes around a week or maybe a few days until the Repository owner gets noticed.

**Comment by Yaser-123** [comment]
A pull request typically takes a few days to a week to get merged, depending on the project's review process, the complexity of the changes, and the maintainers' availability. Smaller, well-documented PRs are often merged faster.

**Comment by Megha-Chakraborty** [comment]
Please merge my pending PRs.
1. https://github.com/Ramsey99/fest-registration/pull/104
2. https://github.com/Shreyaa173/Code-Book/pull/484

Please it's my humble request to merge these pr. I've pulled request a lot earlier but still it not got merged.

**Comment by codeftsaikat** [comment]
Hey, On around, it takes 1 to 7 days to merge a pull request, depending on factors like:

Complexity of Changes: Larger, more complex changes may require thorough review and testing, increasing the time.
Review Process: Teams with active reviewers often merge faster, while smaller teams or those with a backlog might take longer.
Project Activity: Active repositories with well-defined workflows tend to merge quicker than less active or disorganized projects.
Feedback and Iterations: If reviewers request changes, the back-and-forth can add time to the process.
Priority: Critical fixes or features may be merged faster than non-urgent contributions.
For highly active open-source projects, the median time is often 2-4 days, while smaller or less active projects might take longer.
